src/components/MovieCard/index.tsx

```diff
@@ -1,17 +1,39 @@
-import React from "react";
+import React, { useContext } from "react";
 import "./movie-card.scss"
 import { MovieCardProps } from "./interface"
 import { MovieCardContextMenu } from "../MovieCardContextMenu";
+import { actionControlView, ViewContext } from "../../context/ViewContext";
+
+export const MovieCard:React.FunctionComponent<MovieCardProps> = ({ title, genre, link, year, posterUrl, duration, description, rating }) => {
```

Comment:
Wouldn't single prop (e.g. `movieDetails`) be less verbose both in here and `dispatch()` call?